caffeine/build.gradle

```diff
@@ -122,6 +122,7 @@ tasks.named('compileJava').configure {
   dependsOn generateLocalCaches, generateNodes
   finalizedBy compileCodeGenJava
   options.errorprone.disable('CheckReturnValue')
+  options.errorprone.error('CanIgnoreReturnValueSuggester')
```

Comment:
This might go into codeQuality.gradle?
https://github.com/ben-manes/caffeine/blob/d084f2d3f1bb210dcba5b3f054039680178ef707/gradle/codeQuality.gradle#L154-L155

Reply:
Ah, I missed that. Lemme try!

Reply:
This way it also flags some test code. That's fine; will add the annotations there too.